### ISSUE TITLE
How to change loopbackRadius in connectionDragStop event?

### ISSUE THREAD

**Issue by manojp1988** [jsplumb/jsplumb#766]
In my use case, I need to do multiple loop back connections. As you see below in the screenshot, they are overlapped.

Is there any way to change the loop radius when you drag and drop? I'm planning to change the radius when you drag and drop each connection. 

![capture](https://user-images.githubusercontent.com/1810503/42406607-b2fe644c-81c8-11e8-8939-18f62192c23d.PNG)

I'm able to do like below but I have to register connector type and apply them later. Is there any way to pass the loop radius when "connectionDragStop" event fires.
![capture](https://user-images.githubusercontent.com/1810503/42406639-4bdf87b8-81c9-11e8-82df-00f6b8e9db5a.PNG)

**Comment by sporritt** [comment]
no this isnt possible in the api at the moment.

**Comment by manojp1988** [comment]
Oh ok. Thanks.

I would like add drag points in the arrow, so I can adjust the line as per needs. If I have to implement something like this in jsplumb, where should I start.?

Can you guide me?

![screenshot-www draw io-2018 07 07-11-31-36](https://user-images.githubusercontent.com/1810503/42407394-5b70f224-81d9-11e8-9462-78f8e4b11e97.png)

**Comment by sporritt** [comment]
its better to keep each issue focused on one thing.

having said that, there is little point in opening a new issue.  this functionality is going to be added to the toolkit edition (but not the community edition).

**Comment by manojp1988** [comment]
Agreed. But both are connected to the issue I'm having "overlapping arrows".

Thank you. May I know when this new feature will be rolled out in toolkit edition.

**Comment by saeed3e** [comment]
> Oh ok. Thanks.
> 
> I would like add drag points in the arrow, so I can adjust the line as per needs. If I have to implement something like this in jsplumb, where should I start.?
> 
> Can you guide me?
> 
> ![screenshot-www draw io-2018 07 07-11-31-36](https://user-images.githubusercontent.com/1810503/42407394-5b70f224-81d9-11e8-9462-78f8e4b11e97.png)

@sporritt  Is this feature rollout in jsPlumb toolkit edition?

**Comment by sporritt** [comment]
It's not released yet, no. It's kind of finished in terms of the dev work but then there's the rest of the stuff left to go when adding a new feature - documenting it, providing demos, etc.

**Comment by saeed3e** [comment]
@sporritt So, Is there any tentative date by when we can expect the release because this feature is very important in our project.

**Comment by sporritt** [comment]
It's kind of like the proverbial piece of string really. The release date kind of depends on us deciding to prioritise it, which would be helped by either a licensee or a prospective licensee showing some interest.
